src/Approvals/SFA.DAS.Approvals/InnerApi/CommitmentsV2Api/Requests/GetOverlapRequestRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.Approvals.InnerApi.CommitmentsV2Api.Requests
+{
+    public class GetOverlapRequestRequest : IGetApiRequest
+    {
+        public readonly long DraftApprenticeshipId;
+
+        public GetOverlapRequestRequest(long draftApprenticeshipId)
+        {
+            DraftApprenticeshipId = draftApprenticeshipId;
+        }
+
+        public string GetUrl => $"api/overlapping-training-date-request/{DraftApprenticeshipId}/get";
```

Comment:
/get on the end of the route is unnecessary

Reply:
updated to new name